js/crm.ajax.js

```diff
@@ -456,13 +456,13 @@
         var buttonContainers = '.crm-submit-buttons, .action-link',
           buttons = [],
           added = [];
-        $(buttonContainers, $el).find('input.crm-form-submit, a.button').each(function() {
+        $(buttonContainers, $el).find('input.crm-form-submit, a.button, button').each(function() {
           var $el = $(this),
             label = $el.is('input') ? $el.attr('value') : $el.text(),
             identifier = $el.attr('name') || $el.attr('href');
           if (!identifier || identifier === '#' || $.inArray(identifier, added) < 0) {
             var $icon = $el.find('.icon, .crm-i'),
-              button = {'data-identifier': identifier, text: label, click: function() {
+              button = {class: 'crm-button button', 'data-identifier': identifier, text: label, click: function() {
```

Comment:
What is the significance of adding these classes to the popup's buttons? On my sandbox this makes no difference to the appearance. Is this toward further work? If so I'd say only add `crm-button` and not `button` which is a drupal css class not really a civi class (although civi forms do use it but that's not a good pattern to follow).

Reply:
Let's wait until https://github.com/civicrm/civicrm-core/pull/14519 (submitonce) is resolved/merged as this is sort of related - I'll update this PR as I'll need to add the button selector as well to the submitonce function.
I added the button class as well as this helps them render properly when using a bootstrap based theme.

Reply:
I've had another look at this and we don't actually need to set the classes here at all as we can pull them across from the original buttons in formButtons.tpl.  So I've removed this bit.